README.md

```diff
@@ -431,9 +431,9 @@ This is how we build Windows releases.
 Version scheme
 --------------
 
-Minetest doesn't follow semver. Instead, we do something roughly similar to 0.major.minor.
+Minetest follows partially semver.
```

Comment:
Better English is:
'Minetest partially follows semver.'

Reply:
yeah, i'm not very up this morning :)

Reply:
Maybe also specify we use major.minor.patch?
Maybe inform when we started using M.M.P. and that we used 0.M.M up to 0.4? This text specifies what we used in the past too: "Prior to that, we used oldversion-dev.".